ui-tests/src/main/java/com/practicesoftwaretesting/utils/SelectorUtils.java

```diff
@@ -0,0 +1,20 @@
+package com.practicesoftwaretesting.utils;
+
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Selectors.by;
+
+public class SelectorUtils {
+
+    public static By byDataTest(String attributeValue) {
+        return by("data-test", attributeValue);
+    }
+
+    public static By byFor(String attributeValue) {
+        return by("for", attributeValue);
+    }
+
+    public static By byId(String attributeValue) {
```

Comment:
Do not reinvent the bicycle
Read the documentation, please. If you think of any improvement, please check if somebody has already done it.
https://selenide.org/javadoc/current/com/codeborne/selenide/Selectors.html#byId(java.lang.String)

Reply:
It was made by mistake, thanks for comment